src/resolvers/auth.js

```diff
@@ -1,15 +1,43 @@
 const bcrypt = require('bcryptjs');
 const models = require('../models');
+const { sendEmail } = require('../services/sendEmail');
 
 const {
+  authenticateResetToken,
   createNewAccessToken,
   createNewRefreshToken,
+  createNewResetToken,
   addAccessTokenCookie,
   addRefreshTokenCookie,
   clearAccessTokenCookie,
   clearRefreshTokenCookie,
 } = require('../middleware/auth');
 
+const { CLIENT_URL } = require('../config/config');
+
+const createResetTokenedEmail = (resetToken) => {
+  const resetPasswordUrl = `${CLIENT_URL}/resetPassword/${resetToken}`;
+  return {
+    from: 'kevinzhang@uwblueprint.org',
+    subject: `Raising the Roof Password Reset Attempt`,
+    html: `Reset your password here: ${resetPasswordUrl}`,
+  };
+};
+
+const createResetAttemptEmail = () => {
+  return {
+    from: 'kevinzhang@uwblueprint.org',
+    subject: `Raising the Roof Password Reset Attempt`,
+    html: `Someone recently attempted to reset your password at ${CLIENT_URL}, however, you do not have an account with us`,
+  };
```

Comment:
Do we need to add `<a>` tags here to make it a link?

Reply:
I think you should have it, just to be consistent with (from #47)
![image](https://user-images.githubusercontent.com/24748963/114465429-19d97680-9bb5-11eb-93a4-7e8ed7f309bb.png)